### ISSUE TITLE
[Bug]: Missing H2H

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/scores#1706]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐞 Describe the bug

Missing H2H:

https://scores.betarena.com/football/porto-fc-barcelona-18997103

### 🤔 Expected Behavior

There was a match between the teams in 2011.

### What environment is the bug/issue detected in.

- [X] Production (scores.betarena.com)
- [ ] Development

### What browsers are you seeing the problem on?

- [X] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Brave
- [ ] Opera

### What device/platform can the problem be seen?

- [ ] 📱 Mobile
- [ ] 💻 Tablet
- [X] 🖥️ Laptop/Desktop

### Anything else?

It might be a provider issue.

**Comment by migbash** [comment]
**📌 Supporting Data**

```json
[..]
"id": 18997103,
[..]
"localteam_id": 652,
"visitorteam_id": 83,
[..]
```

> **Note**
> When checked for the data, it shows that the `Head-2-Head` for these 2 teams is present from `Sportmonks v2`, however, 
- it appears to be absent on the `betarena_prod/public/football_h2h > (col.) team_ids :: 83,652`. 
- **but** it appears to be present on the `betarena_prod/public/historic_fixtures` data table.

**Comment by migbash** [comment]
> **Note**
> Created offspring task ➤ #<-github-issue-number->.

**Comment by migbash** [comment]
> **Note**
> Already addressed as part of issue ➤ [#249](https://github.com/Betarena/backend_betarena/issues/249).
> Implemented, awaiting `🚀 deployment`.
